bundles/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/configuration/ICloudAccountThingConfiguration.java

```diff
@@ -22,7 +23,7 @@
  */
 @NonNullByDefault
 public class ICloudAccountThingConfiguration {
-    public String appleId;
-    public String password;
-    public Integer refreshTimeInMinutes;
+    public @Nullable String appleId;
+    public @Nullable String password;
+    public @Nullable int refreshTimeInMinutes;
```

Comment:
That won‘t work. Either provide a default and use int or @Nullable and Integer.

Reply:
I stayed with int and provided a default value.